charts/cluster-api-cluster-openstack/README.md

```diff
@@ -49,28 +50,25 @@ spec:
           region: en-west-1
           failureDomain: eu-west-1a
           externalNetworkID: dadfef54-d1c5-447a-8933-f515eeadd822
-        cluster:
-          taints:
-          - key: node.cilium.io/agent-not-ready
-            effect: NoSchedule
-            value: 'true'
         api:
           allowList:
           - 123.45.67.89
           certificateSANs:
           - kubernetes.my-domain.com
         controlPlane:
+          version:  v1.30.2
```

Comment:
Note to self: we should probably be more opinionated here and just set it at the top level, mixed verison clusters seems like a bad idea.